core/src/dird/vbackup.cc

```diff
@@ -541,12 +541,14 @@ void NativeVbackupCleanup(JobControlRecord* jcr, int TermCode, int JobLevel)
  */
 static int InsertBootstrapHandler(void* ctx, int, char** row)
 {
-  JobId_t JobId;
-  int FileIndex;
-  RestoreBootstrapRecord* bsr = (RestoreBootstrapRecord*)ctx;
+  auto* rx = reinterpret_cast<RestoreContext*>(ctx);
```

Comment:
```suggestion
  auto* rx = static_cast<RestoreContext*>(ctx);
```
from and to `void*` should use `static_cast<>`.